.github/workflows/main.yml

```diff
@@ -37,3 +37,25 @@ jobs:
         run: |
           echo $GITHUB_WORKSPACE
           docker run --privileged -e CI=true -t -v $GITHUB_WORKSPACE:/github/workspace docker.pkg.github.com/fwupd/fwupd/fwupd-${{matrix.os}}:latest
+
+  fuzzing:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Build Fuzzers
+      id: build
+      uses: google/oss-fuzz/infra/cifuzz/actions/build_fuzzers@master
+      with:
+        oss-fuzz-project-name: 'fwupd'
+        dry-run: false
+    - name: Run Fuzzers
+      uses: google/oss-fuzz/infra/cifuzz/actions/run_fuzzers@master
+      with:
+        oss-fuzz-project-name: 'fwupd'
+        fuzz-seconds: 300
```

Comment:
so it basically has a 300 second time for every job?

Reply:
Yup, the default was 10 minutes but I thought that was a bit too long to wait for each PR. Less is fine too, even 60 seconds would find any glaring issues.

Reply:
Hear out my idea.  How about if we have 10 minutes on master nightly but then something short only on the code that _changed_?  Is that crazy?

Reply:
>  How about if we have 10 minutes on master nightly

We get several hours on master twice a day; the oss-fuzz bot has been opening AddressSanitizer bugs against fwupd all morning :) The CI version of oss-fuzz is meant as a quick check.

Reply:
I meant to add; if you want cc on those bugs let me know and I'll submit a pr to oss-fuzz so that you're on the list too. Slight warning, the only way to opt out is another PR...

Reply:
>We get several hours on master twice a day; the oss-fuzz bot has been opening AddressSanitizer bugs against fwupd all morning :) The CI version of oss-fuzz is meant as a quick check.

Oh if it's already running like that, yeah I think what you're doing here makes a lot more sense.

>if you want cc on those bugs

You can leave me off for now.